src/matchers/text.rs

```diff
@@ -98,4 +104,14 @@ mod tests {
     fn shellscript() {
         assert!(!is_shellscript(b"#!"));
     }
+
+    #[test]
+    fn plain_text() {
+        assert!(is_text_plain(b"hello, world"));
+        assert!(is_text_plain(b"\xF0\x9F\x92\xA9")); // poop emoji
+        assert!(is_text_plain(b"\xE3\x81\x93")); // Japanese letter
+        assert!(is_text_plain(b"hello\tworld\nhow\r\nare you?"));
+        assert!(!is_text_plain(b"\x08")); // backspace
+        assert!(!is_text_plain(b"OK\x08")); // backspace
```

Comment:
Combined plain text with control character to catch people who struggle with `all` vs `any` functions (like me, it turns out).